backend/models/organizations.js

```diff
@@ -1,29 +1,22 @@
 const mongoose = require("mongoose");
 
 const user = new mongoose.Schema({
-  userId: {
-    type: String
-  },
-  admin: {
-    type: Boolean
+  id: {
+    type: mongoose.SchemaTypes.ObjectId,
+    unique: true
   }
 });
 
-//Currently we only have one role: reviewer.
-//In the future we may want to have "spectators" for example
 const program = new mongoose.Schema({
   id: {
-    type: String,
+    type: mongoose.SchemaTypes.ObjectId,
     unique: true
   }
 });
 
-//Programs[] is a list of program IDs that the organization has has been assigned to.
-//They can only view these programs (unless admin).
-
 const organizationSchema = new mongoose.Schema(
   {
-    users: {
+    admins: {
```

Comment:
How do we plan on storing non-admin users in the organization schema? This doesn't look to take that into account.

Reply:
Non-admin users can be looked up through the programs under the org!
Not sure if there's a better way to structure that.

Reply:
I guess my answer to this would depend on the following question. Can a non-admin user be on an organization scope? Or will they only be added on a program-to-program basis? And conversely, will an admin user need to be added to the program's users array or is it implied that an admin user in org has access to all programs?

Reply:
Users who aren't org admins will be added on a program-to-program basis! From the designs, "a user of the organization" is equivalent to "a user who's part of any program under the organization".

And yes, it's implied that all org admins automatically have access to all programs under that org